pennylane/transforms/qcut.py

```diff
@@ -18,10 +18,137 @@
+
+    **Example**
+
+    Consider the following circuit with a manually-placed wire cut:
+
+    .. code-block:: python
+
+        from pennylane.transforms import qcut
+
+        wire_cut = qml.WireCut(wires=0)
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(0.4, wires=0)
+            qml.apply(wire_cut)
+            qml.RY(0.5, wires=0)
+            qml.expval(qml.PauliZ(0))
+
+    We can find the circuit graph and remove the wire cut node using:
+
+    >>> graph = qcut.tape_to_graph(tape)
+    >>> qcut.replace_wire_cut_node(wire_cut, graph)
+    """
+    predecessors = graph.pred[node]
+    successors = graph.succ[node]
+
+    predecessor_on_wire = {}
+    for op, data in predecessors.items():
+        for d in data.values():
+            wire = d["wire"]
+            predecessor_on_wire[wire] = op
+
+    successor_on_wire = {}
+    for op, data in successors.items():
+        for d in data.values():
+            wire = d["wire"]
+            successor_on_wire[wire] = op
+
+    order = graph.nodes[node]["order"]
+    graph.remove_node(node)
+
+    for wire in node.wires:
+        predecessor = predecessor_on_wire.get(wire, None)
+        successor = successor_on_wire.get(wire, None)
+
+        meas = MeasureNode(wires=wire)
+        prep = PrepareNode(wires=wire)
+
+        # We are introducing a degeneracy in the order of the measure and prepare nodes
+        # here but the order can be inferred as MeasureNode always precedes
+        # the corresponding PrepareNode
+        graph.add_node(meas, order=order)
+        graph.add_node(prep, order=order)
+
+        graph.add_edge(meas, prep, wire=wire)
+
+        if predecessor is not None:
+            graph.add_edge(predecessor, meas, wire=wire)
+        if successor is not None:
+            graph.add_edge(prep, successor, wire=wire)
+
+
+def replace_wire_cut_nodes(graph: MultiDiGraph):
+    """
+    Replace all :class:`~.WireCut` nodes in the graph with
+    :class:`~.MeasureNode` and :class:`~.PrepareNode`.
+
+    Args:
+        graph (MultiDiGraph): The graph containing the :class:`~.WireCut` nodes
+        to be replaced
+
+    **Example**
+
+    Consider the following circuit with a manually-placed wire cut:
+
+    .. code-block:: python
+
+        from pennylane.transforms import qcut
+
+        wire_cut_0 = qml.WireCut(wires=0)
+        wire_cut_1 = qml.WireCut(wires=1)
+        multi_wire_cut = qml.WireCut(wires=[0, 1])
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(0.4, wires=0)
+            qml.apply(wire_cut_0)
+            qml.RY(0.5, wires=0)
+            qml.apply(wire_cut_1)
+            qml.CNOT(wires=[0, 1])
+            qml.apply(multi_wire_cut)
+            qml.RZ(0.6, wires=1)
+            qml.expval(qml.PauliZ(0))
+
+    We can find the circuit graph and remove all the wire cut nodes using:
+
+    >>> graph = qcut.tape_to_graph(tape)
+    >>> qcut.replace_wire_cut_nodes(wire_cut, graph)
```

Comment:
```suggestion
    >>> qcut.replace_wire_cut_nodes(graph)
```
Always good to check that the code runs locally :+1: